.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           - test:parseui:all
           - test:parse_live_query:all
           - build:starters
-      fail-fast: false
+      fail-fast: true
```

Comment:
We usually don't want to fail fast in order to see the impact of all issues at once. Is there a reason we'd want to change that in this repo?

Reply:
If a test or build fails artifact logs are produced so we can debug them locally. Unfortunately artifacts aren't published until all jobs are finished. I'm adding fast fail here because I don't want to wait 30 minutes for the CI to finish.

Reply:
Got it, let's fail fast then